benchmarks/viscoelastic_beam_modified/10km_opentopbot.prm

```diff
@@ -0,0 +1,196 @@
+# This parameter file modifies a 2D benchmark for viscoelastic deformation
```

Comment:
If you are only changing a few parameters relative to the viscoelastic bending beam benchmark, you can use the following line to derive from that parameter and only include modified parameters here:
`include $ASPECT_SOURCE_DIR/benchmarks/viscoelastic_bending_beam/viscoelastic_bending_beam.prm`

It seems like quite a few things are duplicate, so maybe this would be a good strategy here? This practice also ensure that the viscoelastic_bending_beam test will likely catch anything that breaks in this parameter file as well.

Reply:
Done!